torch/nn/parallel/distributed.py

```diff
@@ -153,11 +153,22 @@ class DistributedDataParallel(Module):
 
     Args:
         module (Module): module to be parallelized
-        device_ids (list of int or torch.device): CUDA devices (default: all devices)
-        output_device (int or torch.device): device location of output (default: device_ids[0])
+        device_ids (list of int or torch.device):  CUDA devices. This should
```

Comment:
Extra whitespace -- is this intentional?

Reply:
no, not intentional. I will edit, thanks!